documentation/operators/src/nodes/maintenance.md

```diff
@@ -0,0 +1,699 @@
+
+Use the following command to get `tmux`.  
+
+Platform|Install Command
+---|---
+Arch Linux|`pacman -S tmux`
+Debian or Ubuntu|`apt install tmux`
+Fedora|`dnf install tmux`
+RHEL or CentOS|`yum install tmux`
+macOS (using Homebrew|`brew install tmux`
+macOS (using MacPorts)|`port install tmux`
+openSUSE|`zypper install tmux`
+  
+In case it didn't work for your distribution, see how to build `tmux` from [version control](https://github.com/tmux/tmux#from-version-control).  
+
+**Running tmux**
+
+No when you installed tmux on your VPS, let's run a mixnode on tmux, which allows you to detach your terminal and let your `<NODE>` run on its own on the VPS.
+
+* Pause your `<NODE>`
+* Start tmux with the command 
+```
+tmux
+```
+* The tmux terminal should open in the same working directory, just the layout changed into tmux default layout.
+* Start the `<NODE>` again with a command:
+```
+./<NODE> run --id <YOUR_ID>
+```
+* Now, without closing the tmux window, you can close the whole terminal and the `<NODE>` (and any other process running in tmux) will stay active.
+* Next time just start your teminal, ssh into the VPS and run the following command to attach back to your previous session:
+```
+tmux attach-session
+```
+* To see keybinding options of tmux press `ctrl`+`b` and after 1 second `?`
+
+#### systemd
+
+Here's a systemd service file to do that:
+
+##### For Mixnode
+
+```ini
+[Unit]
+Description=Nym Mixnode ({{mix_node_release_version}})
+StartLimitInterval=350
+StartLimitBurst=10
+
+[Service]
+User=<USER>
+LimitNOFILE=65536
+ExecStart=/home/<USER>/<PATH>/nym-mixnode run --id <YOUR_ID>
+KillSignal=SIGINT
+Restart=on-failure
+RestartSec=30
+
+[Install]
+WantedBy=multi-user.target
+```
+
+* Put the above file onto your system at `/etc/systemd/system/nym-mixnode.service`.
+
+##### For Gateway
+
+```ini
+[Unit]
+Description=Nym Gateway ({{platform_release_version}})
+StartLimitInterval=350
+StartLimitBurst=10
+
+[Service]
+User=nym
+LimitNOFILE=65536
+ExecStart=/home/<USER>/<PATH>/nym-gateway run --id <YOUR_ID>
+KillSignal=SIGINT
+Restart=on-failure
+RestartSec=30
+
+[Install]
+WantedBy=multi-user.target
+```
+
+* Put the above file onto your system at `/etc/systemd/system/nym-gateway.service`.
+
+##### For Network requester
+
+```ini
+[Unit]
+Description=Nym Network Requester ({{platform_release_version}})
+StartLimitInterval=350
+StartLimitBurst=10
+
+[Service]
+User=nym # replace this with whatever user you wish
+LimitNOFILE=65536
+# remember to add the `--enable-statistics` flag if running as part of a service grant and check the path to your nym-network-requester binary
+ExecStart=/home/nym/nym-network-requester run --id <YOUR_ID>
+KillSignal=SIGINT
+Restart=on-failure
+RestartSec=30
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Now enable and start your requester:
```

Comment:
why only requester? these commands should be generic imo - replacing `network-requester.service` with `<NODE>.service` or something